esp/src/eclwatch/ECLPlaygroundWidget.js

```diff
@@ -111,7 +111,7 @@ define([
                 var logicalCluster = context.targetSelectWidget.selectedTarget();
                 var submitBtn = registry.byId(context.id + "SubmitBtn");
                 var publishBtn = registry.byId(context.id + "PublishBtn");
-                if (logicalCluster.QueriesOnly) {
+                if (logicalCluster.QueriesOnly || logicalCluster.Type === "roxie") {
```

Comment:
This should really be in a diff PR (especially if the tests are flakey and this doesn't get pulled...)